.travis.yml

```diff
@@ -50,27 +51,29 @@ before_install:
 - cp config-server.js.example config-server.js
 - cp test/config.yml.example test/config.yml
 before_script:
-- docker-compose build
-- docker-compose -f docker-compose.yml -f docker-test.yml up -d
+- TRAVIS_JOB_NAME=$TRAVIS_JOB_NAME ./build.sh
```

Comment:
I didn't understand this line... why we need to redefine this variable? If needed, can we assign to a new variable?